lang/cs/Org.Apache.REEF.Driver/Constants.cs

```diff
@@ -32,10 +32,10 @@ public class Constants
         /// <summary>
         /// The class hierarchy file from .NET.
         /// </summary>
-        public const string ClassHierarachyBin = "clrClassHierarchy.bin";
```

Comment:
Public constant in non-example non-test code => needs to go through "Obsolete in 0.14" path